### DISCUSSION TITLE
Control rendering update

### DISCUSSION THREAD

**Discussion by Aleks59** [MaxRobinsonTheGreat/EvolutionSimulatorV2#132] [Ideas and Suggestions]
Add control to rendering update, so it's not "draw every frame" or "don't draw at all" but something between. So "player" still can look at the field, not just numbers, and it will not significantly increase compute time for rendering.
It can be just slider with textbox with frequency or period.

**Comment by theomgdev** [comment]
https://cklifeengine.netlify.app

I have already written the code for it. Just use the skip frame option at the left bottom of the screen.
Max also liked this feature, but he asked me to sync this feature to 60 FPS (minimum lag-free human perception) for a setting above 60 FPS, and this is very difficult.

**Reply by SuperSpaceEye** [reply]
Why don't you use timer to track time between updates? Something like:
```
period = 1/60
time = timer.now()
while true:
  if (timer.now() - time > period):
    make_image()
    time = timer.now()
```
